### PR TITLE
Support group_add param in host config

### PR DESCRIPTION
**Pull request by shin-**
Related to #765

### CODE REVIEW

**Comment by dnephin** [comment]
LGTM

File: docker/utils/utils.py [hunk 1]
```diff
@@ -509,6 +509,13 @@ def create_host_config(
     if devices:
         host_config['Devices'] = parse_devices(devices)
 
+    if group_add:
+        if compare_version(version, '1.20') < 0:
```
Comment (aanand) [inline docker/utils/utils.py]: I think this should have been `>`, not `<` - if I set the DEFAULT_DOCKER_API_VERSION to '1.21', then this error gets raised.

Comment (shin-) [inline docker/utils/utils.py]: yup, I think I fixed it afterwards but it may be in one of the pending PRs. Let me find it.

Comment (shin-) [inline docker/utils/utils.py]: https://github.com/docker/docker-py/commit/6cf7758f25de72a3f53528de0a376eadba902ae9#diff-193b0cbc51c3ae87519912dd6aeadec4R522

**Pull request merged**